### DISCUSSION TITLE
fil机房建设、专业节点搭建，定制APP，定制平台系统开发

### DISCUSSION THREAD

**Discussion by suishibao** [filecoin-project/community-china#204] [应用开发]
小型机房建设，专业节点搭建、分币系统APP定制全套filecoin,包括测ceph存储技术全部免费奉送，长期维护只抽取5%服务费！

**Comment by suishibao** [comment]
vx18620044433

**Comment by yunfens** [comment]
对矿工而言，降低维护成本，提高产矿效率才是王道，现在对很多挖矿的人，特别是不懂技术的人，存储用专业的，费用也很高，养几个技术维护的成本也很高，而且出了问题他们也担不起任何赔偿责任，其实这个东西没那么高深，出点小问题也没那么可怕，之所以高深，是因为你不懂，其实只要一个大专生能全部搞定，从设备采购，网络配置，节点搭建，后期运维。我的使命就是为降低矿工成本而来，有兴趣可以交流！

**Comment by yunfens** [comment]
yunfen=suishibao=vx18620044433
